share/completions/fab.fish

```diff
@@ -0,0 +1,75 @@
+# tasks 
+for task in (fab -l)
+    set -l _matched (string match -r "^ +([^ ]*) *([^ ]?.*)" $task)
+
+    if test -n "$_matched"
+        set -l _task_name $_matched[2]
+        set -l _task_desc $_matched[3]
+        test -n "$task_desc"; or set -l _task_desc "No description"
+        complete -f -c fab -a "$_task_name" -d "$_task_desc"
```

Comment:
That's what I wanted to know!!!!

improved 😄 

Move `for ~ end` to function `__fish_complete_fab_tasks `.
And 
Change like following:
`complete -x -c fab -a "(__fish_complete_fab_tasks)"`